ci/jjb/jobs/pulp3-smash-runner.yaml

```diff
@@ -122,7 +124,15 @@
 
             echo "Run pytest"
             set +e
-            py.test -v --color=yes --html=report.html --self-contained-html --junit-xml=junit-report.xml pulpcore/pulpcore/tests/functional pulp_file/pulp_file/tests/functional pulp_rpm/pulp_rpm/tests/functional pulp_docker/pulp_docker/tests/functional
+            py.test -v --color=yes \
+              --html=report.html --self-contained-html \
+              --junit-xml=junit-report.xml \
+              pulpcore/pulpcore/tests/functional \
+              pulp_file/pulp_file/tests/functional \
+              pulp_rpm/pulp_rpm/tests/functional \
+              pulp_docker/pulp_docker/tests/functional \
+              pulp-certguard/pulp_certguard/tests/functional \
```

Comment:
@bmbouter hey, look, that is why I asked you about renaming the `pulp-certguard` repository to `pulp_certguard` there is a pattern here on the use of `_` and because of the `-` on certguard it is more difficult to build those parameters based on the list of installed plugins.

Reply:
I see what you were talking about.